project/ValidatePullRequest.scala

```diff
@@ -209,4 +208,42 @@ object ValidatePullRequest extends AutoPlugin {
     // add reportBinaryIssues to validatePullRequest on minor version maintenance branch
     validatePullRequest <<= validatePullRequest.dependsOn(reportBinaryIssues)
   )
+
+  def whenValidatingPr(task: TaskKey[_]) =
+    validatePullRequest := {
+      (projectBuildMode in ValidatePR).value match {
+        case BuildSkip => // do not run the additional task if project is skipped during pr validation
+        case _ => task.value
```

Comment:
Hmm, I think you're right Roland but we'll need to verify to be sure I guess. In the validator itself it's fine as `Def.taskDyn` is used https://github.com/akka/akka/blob/master/project/ValidatePullRequest.scala#L201
In taskDyn when you inspect a task you can't see what it depends on, because the dependencies only appear once you run it, that's what we need here as well.